### ISSUE TITLE
WebRTC/Offer stream not found

### ISSUE THREAD

**Issue by cbroissa-github** [AlexxIT/WebRTC#522]
Hello,
With the new version 3.2.0 on Home Assistant, my cam doesn't work with the error WebRTC/Offer stream not found.
Any idea ?
Thank you.

**Comment by uowis** [comment]
Same error nothing in log.

**Comment by AlexxIT** [comment]
You need to provide all the details of how your streams are set up.

**Comment by uowis** [comment]
```
type: custom:webrtc-camera
url: rtsp://192.xxxx.xxxx.xx:8080/h264_ulaw.sdp
```
Android ip web cam
RTSP work

debug : 
```
2023-07-11 14:03:55.782 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:03:55.782[0m [32mINF[0m go2rtc version 1.6.0 linux/arm64
2023-07-11 14:03:55.786 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:03:55.784[0m [32mINF[0m [api] listen [36maddr=[0m:1984
2023-07-11 14:03:55.787 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:03:55.786[0m [32mINF[0m [rtsp] listen [36maddr=[0m:8554
2023-07-11 14:03:55.788 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:03:55.787[0m [32mINF[0m [webrtc] listen [36maddr=[0m:8555/tcp
2023-07-11 14:03:55.802 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:03:55.800[0m [32mINF[0m [srtp] listen [36maddr=[0m:8443
2023-07-11 14:04:17.643 DEBUG (MainThread) [custom_components.webrtc] New client: {'authSig': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiI5N2M2NDQ2MDVjYzY0YjE3OTRkODY2MDY0Yzg4NGMxYSIsInBhdGgiOiIvYXBpL3dlYnJ0Yy93cyIsInBhcmFtcyI6W10sImlhdCI6MTY4OTA3NzA1NiwiZXhwIjoxNjg5MDc3MDg2fQ.63-Up8rLua9elbyIlSeZAaK2ng7R97Xg7zFhTUd5INA', 'url': 'rtsp://xxxxxxx:xxxx/h264_ulaw.sdp'}
2023-07-11 14:09:38.465 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:09:38.460[0m [31mWRN[0m [1mgithub.com/AlexxIT/go2rtc/internal/hass/hass.go:54[0m[36m >[0m [36merror=[0m[31m"hass: wrong type: auth_invalid"[0m
2023-07-11 14:10:26.662 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:10:26.661[0m [31mWRN[0m [streams] json locked
2023-07-11 14:10:27.517 DEBUG (webrtc) [custom_components.webrtc.utils] [90m14:10:27.515[0m [1m[31mERR[0m[0m [1mgithub.com/AlexxIT/go2rtc/internal/mjpeg/init.go:178[0m[36m >[0m [36merror=[0m[31m"dial tcp 192.168.1.113:554: connect: no route to host"[0m
```

**Comment by cbroissa-github** [comment]
type: custom:webrtc-camera
url: rtsp://192.xx.xx.xx/ch0_0.h264
ui: true

**Comment by andonevris** [comment]
Same issue here, for me it is because of go2rtc 1.6.0, reverting to go2rtc 1.5.0 brings the streams back.

As a temporary workaround I used a crude hack, in the Home Assistant config folder I replaced go2rtc 1.6.0 with 1.5.0 but that will get overwritten so you need to rename the 1.5.0 to 1.6.0 as a spoof, luckily it looks like the integration only checks the name so it falls for the spoof.

**Comment by uowis** [comment]
i have delete go2rc and reboot it's ok for me now

**Comment by bkbartk** [comment]
I have a similar issue (only is says "mse" instead of webrtc/offer)
![image](https://github.com/AlexxIT/WebRTC/assets/17070805/4cc2f666-b152-4992-903c-24d5e827642b)
reverting to v 3.1.1 helped.
I noticed the correct version of gotortw (1.5.0 or 1.6.0) is automatically downloaded.

**Comment by andonevris** [comment]
> Bad idea to use the new version of the integration with the old go2rtc. There may be confusing errors.

Yeah for sure, like I said just a crude hack to get the streams back for now

**Comment by Shivy011** [comment]
> Same issue here, for me it is because of go2rtc 1.6.0, reverting to go2rtc 1.5.0 brings the streams back.
> 
> As a temporary workaround I used a crude hack, in the Home Assistant config folder I replaced go2rtc 1.6.0 with 1.5.0 but that will get overwritten so you need to rename the 1.5.0 to 1.6.0 as a spoof, luckily it looks like the integration only checks the name so it falls for the spoof.
> 
> My setup is using basic webrtc camera setup on home assistant installed via HACS connecting to Unifi Protect cameras
> 
> type: custom:webrtc-camera url: rtspx://IP ADDRESS/random string provided by camera UI muted: true background: true

Hi could you advise how to do this?
I cannot find the go2rtc file in cofig

**Comment by andonevris** [comment]
> > Same issue here, for me it is because of go2rtc 1.6.0, reverting to go2rtc 1.5.0 brings the streams back.
> > As a temporary workaround I used a crude hack, in the Home Assistant config folder I replaced go2rtc 1.6.0 with 1.5.0 but that will get overwritten so you need to rename the 1.5.0 to 1.6.0 as a spoof, luckily it looks like the integration only checks the name so it falls for the spoof.
> > My setup is using basic webrtc camera setup on home assistant installed via HACS connecting to Unifi Protect cameras
> > type: custom:webrtc-camera url: rtspx://IP ADDRESS/random string provided by camera UI muted: true background: true
> 
> Hi could you advise how to do this? I cannot find the go2rtc file in cofig

No need to do this anymore, the issue is resolved.

I deleted the web rtc camera integration and the hacs install then re-installed from scratch and all working now.

**Comment by Shivy011** [comment]
> > > Same issue here, for me it is because of go2rtc 1.6.0, reverting to go2rtc 1.5.0 brings the streams back.
> > > As a temporary workaround I used a crude hack, in the Home Assistant config folder I replaced go2rtc 1.6.0 with 1.5.0 but that will get overwritten so you need to rename the 1.5.0 to 1.6.0 as a spoof, luckily it looks like the integration only checks the name so it falls for the spoof.
> > > My setup is using basic webrtc camera setup on home assistant installed via HACS connecting to Unifi Protect cameras
> > > type: custom:webrtc-camera url: rtspx://IP ADDRESS/random string provided by camera UI muted: true background: true
> > 
> > 
> > Hi could you advise how to do this? I cannot find the go2rtc file in cofig
> 
> No need to do this anymore, the issue is resolved.
> 
> I deleted the web rtc camera integration and the hacs install then re-installed from scratch and all working now.

Brilliant thanks, I had not bothered to check if there was an update for this
